### PR TITLE
[JENKINS 62750] Transform rss bar items into transparent buttons

### PR DESCRIPTION
**Pull request by fqueiruga**
See [JENKINS-62750](https://issues.jenkins-ci.org/browse/JENKINS-62750).

This PR reworks the RSS bar to make the links transparent buttons. It also makes the items more accessible by combining the icons and text into a single link.
As a side effect, this PR also adds theming support for transparent buttons, thus opening core support for https://github.com/jenkinsci/dark-theme-plugin/issues/105.

<details>
<summary>Screenshots</summary>

Home screen
Captura de pantalla 2020-06-22 a las 10 20 28

Log recorders
Captura de pantalla 2020-06-22 a las 20 04 50

</details>

### Proposed changelog entries

* Entry 1: JENKINS 62750, Restyling and improved accessibility for the RSS feed bar.
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@timja 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: war/src/main/less/abstracts/colors.less [hunk 1]
```diff
@@ -103,6 +103,8 @@
   --btn-secondary-bg: var(--btn-text-color);
   --btn-secondary-border: #9ba7af;
   --btn-text-color: var(--white);
+  --btn-link-bg--hover: #f8f8f8;
+  --btn-link-bg--active: #eaeff2;
```
Comment (fqueiruga) [inline war/src/main/less/abstracts/colors.less:107]: This will enable theming support for the background

**Comment by timja** [comment]
Changes here look good, tested locally

Is the rss feed design intentionally not updating the log recorders page?

http://localhost:8080/jenkins/log/all

image

**Comment by fqueiruga** [comment]
I wasn't aware of that page, do you think it's worth updating in the same way?

**Comment by timja** [comment]
not sure if anyone even uses it tbh...

If it doesn't take long would be good to be consistent

**Comment by fqueiruga** [comment]
Done, updated the screenshots

**Review by timja — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]

**Comment by timja** [comment]
Thanks for the PR!

This PR will be merged after 24 hours if there's no negative feedback

**Review by res0nance — APPROVED** [review]

**Review by alecharp — APPROVED** [review]

**Pull request merged**